client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md

```diff
@@ -13,63 +13,68 @@ MongoDB is a database that stores data records (documents) for use by an applica
-- Fill the registration form with your information and send it.
-- You should be logged into your new, unverified account.
-- In the top of the screen, a message should appear asking to send you an e-mail for account verification. Send and confirm it.
-- After you confirm your account, click **Create new** in the *MongoDB Deployments* section.
+- Click the <a href='https://cloud.mongodb.com/user#/atlas/login' target='_blank' rel='no-follow'>Sign In</a> button in the top right corner to open the registration page.
+- Click the <a href='https://cloud.mongodb.com/user#/atlas/register/accountProfile' target='_blank' rel='no-follow'>Register for a new account</a> link at the bottom of the sign in page.
+- Fill the registration form with your information and press **continue**.
+- You should now be logged into your new account and see a modal with a green **Build my first cluster** button, click on it.
 
-### Create a free online database.
+### Create a new cluster
 
-Now we are going to create the actual database you are going to be using.
+- Go through the steps of building your first cluster by following the instructions they provide and clicking next after each step.
+    - **Choose your cloud provider and region**, you can leave this as the default provided (typically AWS).
+    - **customize your cluster's specs**, you can also leave this as the default provided, `M0 Sandbox (Shared RAM, 512 MB Storage) Encrypted`.
+    - **Give your cluster a name**, you can also leave this as the default provided, `Cluster 0`.
+- Now click the green **Create Cluster** button at the bottom of the screen and verify the image captions they provide. 
+- You should now see the message, `Your cluster is being created -
+New clusters take between 7-10 minutes to provision.` Wait until the cluster is created before going to the next step.
 
-- Choose a *Cloud Provider* from the available list.
-- Select the *Sandbox* plan type, which is the only one with no cost, and press **Continue**.
-- Select a region for your Sandbox, from the available list, and press **Continue**.
-- Input a name for your database. This name will be used in the URI for your database. After that, press **Continue**.
+### Create a new admin user on the database.
 
-A summary of all your choices should appear, allowing you to change any information provided in the previous steps. Press **Submit Order** to confirm the information.
+- You should be able to see the green **Get Started** button on the bottom left of your screen, you can click this button to see at which step of the process your are in, if you click on it now, you can see the next step is to **Create your first database user**, go ahead and click on that step.
+    - Follow the instructions by clicking on the `Security` tab.
+    - Click on the green **ADD NEW USER** button.
+    - Enter a user name and password and then select **Atlas Admin** under user privileges, remember to store your username and password somewhere safe.
```

Comment:
```suggestion
    - Enter a user name and password and then select **Read and write to any database** under user privileges, remember to store your username and password somewhere safe.
```
No need for admin privileges here, the projects aren't going to need them.

Reply:
Applied the changes per gitter conversation.